packages/react-microlink/src/components/Card/CardContent.js

```diff
@@ -43,13 +43,10 @@ const Description = styled(ClampLines)`
   margin: auto 0;
   line-height: 18px;
 
-  > div {
-    overflow: hidden;
-    text-overflow: ellipsis;
-  }
-
```

Comment:
These are only necessary for "mobile" now - react-clamp-lines takes care of everything when multi-line.